ivy/functional/frontends/paddle/vision/transforms.py

```diff
@@ -135,3 +135,10 @@ def vflip(img, data_format="CHW"):
     elif data_format.lower() == "hwc":
         axis = -3
     return ivy.flip(img, axis=axis)
+
+@with_unsupported_device_and_dtypes(
+    {"2.5.1 and below": ("float16", "float32", "int16", "int8", "uint8")}, "paddle"
+)
+@to_ivy_arrays_and_back
+def crop(img, top, left, height, width):
+    return img[top:top+height, left:left+width]
```

Comment:
Thanks for the update! Now the only thing remaining is to resolve the lint errors showing up in the CI check at the bottom of the PR.

Reply:
Thank for your reviewing and help so far! I am struggling to understand how to resolve these errors. More specifically, I am lost on how to find sufficient documentation that fully explains the parameters within the decorator '@handle_frontend_test' in order to make sure that the code there is correct as well as understanding how to navigate and debug the code based on the lint errors provided. Your help would be greatly appreciated!

Reply:
Resolving the lint errors is actually much simpler than what you are suggesting! 
Here what we can see in the CI check below:
![image](https://github.com/unifyai/ivy/assets/111577222/750d3b58-1aa3-44c2-b759-228306fc135d)
The red and green highlights indicate the before and after of the changes you need to make. So, just add spaces in the index of `return img[top:top+height, left:left+width]` and add a `,` after `max_dim_size=100`. 
Lint errors are basically formatting errors in the code you are submitting.